.github/workflows/lifecycle.yaml

```diff
@@ -0,0 +1,15 @@
+name: Bucket Lifecycle Tests
```

Comment:
@baum, I tend to agree with @liranmauda. you can add the tests to `src/test/unit_tests/index.js` instead of running them in a separate action.
we might want to eventually break our tests into different actions, but this is out of the scope of this PR